libraries/unit_converter.rb

```diff
@@ -0,0 +1,45 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# File Name:: unit_converter.rb
+#
+# Module contains different functions used to manipulate the units of file sizes.
+#
+module CernerSplunk
+  SIZE_SCALE = %w(KB MB GB TB).freeze
+  # Methods converts file sizes in KB, MB, GB and TB into Bytes.
+  def self.convert_to_bytes(string) # rubocop:disable CyclomaticComplexity
```

Comment:
Looks better than what I have.  Still working on the regex to understand everything 
However updated the unit_converter.rb and added spec to confirm it works.
87f5983cc3cee5ee752edb9e2576bc0a8dcfd6a5